crates/amaru/tests/conformance/stake-distributions/Makefile

```
@@ -70,36 +56,40 @@ check-aws-env:
-	tar) \
-		aws s3 cp --endpoint-url "$(ENDPOINT)" "s3://$(BUCKET)/$(ARCHIVE)" "$(ARCHIVE)" \
-		;; \
-	xz) \
-		aws s3 cp --recursive --endpoint-url "$(ENDPOINT)" --exclude "*" --include "epoch_*.json.xz" "s3://$(BUCKET)/" . \
-		;; \
-	*) \
-		echo "WHAT must be one of: tar, xz"; \
-		exit 1; \
-		;; \
-	esac
+download: check-network-dir check-aws-env ## Download missing 'epoch_*.json.zst' fixtures from the network-specific S3 bucket
```

checkmake (0.3.2)

[warning] Target body for "download" exceeds allowed length of 5 lines (17).

(maxbodylength)